p2p/validation/msg_validation.go

```diff
@@ -14,15 +14,23 @@ import (
 // MsgValidatorFunc represents a message validator
 type MsgValidatorFunc = func(ctx context.Context, p peer.ID, msg *pubsub.Message) pubsub.ValidationResult
 
-func MsgValidation(runner ssv.Runner) MsgValidatorFunc {
+type MessageValidator interface {
+	ValidateSignedSSVMessage(signedSSVMessage *types.SignedSSVMessage) error
+	ValidateSignature(signedSSVMessage *types.SignedSSVMessage) error
+}
```

Comment:
Add comments:
1. Open a github issue that we need to add spec for MsgValidation and reference for `ValidateSignedSSVMessage`. Comment with the link to the issue.
2. For `ValidateSignature` comment that it is an RSA sig with PKCSv1.5 padding. You can link to the [RFC](https://datatracker.ietf.org/doc/html/rfc8017#section-8.2.2)... I think the link I gave is good, you can double-check 😛